benchmarks/intmm.py

```diff
@@ -0,0 +1,91 @@
+import torch
+import pathlib
+import argparse
+import torch.nn.functional as F
+import itertools
+import torch.utils.benchmark as benchmark
+import math
+import csv
+from torchao.kernel.intmm_triton import int_matmul, int_scaled_matmul
+
+torch._dynamo.config.cache_size_limit = 128
+torch._dynamo.config.accumulated_cache_size_limit = 128
+
+dtype = torch.float16
+device = "cuda"
+
+
+
+def benchmark_in_ms(warmup, iters, f, *args, **kwargs):
```

Comment:
put this in benchmark util instead?

Reply:
Once I add the next benchmark for weight only